src/components/Popups/PopupHighlight.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+import IconHighlightTextAnnotation from 'box-ui-elements/es/icons/annotations/IconHighlightTextAnnotation';
```

Comment:
Does this cause any issues with duplicate styles in the host application? We had that problem previously and had to remove all imported BUIE components.

Reply:
I don't think it will cause any issues with styles. `IconHighlightTextAnnotation` only imports `AccessibleSVG` and `AccessibleSVG` does not have a stylesheet.